docs-md/concepts/materialized-views.md

```diff
@@ -1,16 +1,53 @@
 ---
 
-One of the main purposes of tables in any database is to enable efficient queries over the data. We’ve seen how ksqlDB stores events immutability in Kafka with a simple key/value model. How can queries be made efficient under this model? The answer is by leveraging materialized views. How this works requires a bit of an explanation.
+In any database, one of the main purposes of a table is to enable efficient
+queries over the data. ksqlDB stores events immutably in {{ site.aktm }} by
+using a simple key/value model. But how can queries be made efficient under
+this model? The answer is by leveraging materialized views.
 
-Streams and tables share an intimate relationship. A stream is a sequence of events from which a table can be derived. For example, a sequence of credit scores for a loan applicant can change over time. The sequence of credit scores is a stream. But that stream be interpreted as a table to describe the applicant’s current credit score. On the flip side, the table that represents current credit stores is really two things: the current credit scores, and also the sequence of changes to the credit scores for each applicant. This is a profound realization, and much has been written on this stream/table duality.  Many databases throw away the series of changes to a table and only keep the current state around, thus breaking the natural duality.
+Streams and tables are closely related. A stream is a sequence of events that
+you can derive a table from. For example, a sequence of credit scores for a
+loan applicant can change over time. The sequence of credit scores is a stream.
+But this stream be interpreted as a table to describe the applicant's current
+credit score.
 
-For our purposes, the important thing that enables efficient queries is access to the current state of applied changes. This is more commonly known as a materialized view. 
+Conversely, the table that represents current credit stores is really two
+things: the current credit scores, and also the sequence of changes to the
+credit scores for each applicant. This is a profound realization, and much has
+been written on this stream/table duality.  Many databases throw away the
+series of changes to a table and only keep the current state around, which
+breaks the natural duality.
 
-More precisely, a materialized view is the result of incrementally-updated aggregation over a stream. When a new event is integrated, the current state of the view evolves into a new state. This transition happens by applying the aggregation function that defines the view with the current state and the new event. In this way, a view is never “fully recomputed” when new events arrive. It simply incrementally adjusts to account for the new information. Thus, queries against materialized views are highly efficient because they simply look up the value of the current state. They do no further work.
+What enables efficient queries is access to the current state of applied

```

Comment:
I don't understand what you mean here by "access to the current state of applied changes". The benefit of materialized views is evaluating a query on the delta (only the changes) instead of on the entire table.

Reply:
Yeah I don't think I wrote this particularly clearly. Can you help patch the warts?